registry-analysis/Readme.md

```diff
@@ -408,11 +408,15 @@ It is unrealistic to incorporate a complete list of bindings into a REC document
 To choose a mechanism and the rules, we document our requirements and expectations below:
 
 - A binding should be written by people with a good understanding of the protocol, media type (or similar), and not necessarily the TD Editors. This includes people and organizations inside and outside of the WoT WG.
+  - TODO: Ask existing registry "managers" about this kind of a direction.
 - A binding should correspond to a TD specification version.
 - Association of a binding to the TD specification should be confirmed (or allowed) by the TD task force. In other words, any person cannot say that this binding can be used with the TD version X.
 - It should be possible to register, update, or remove the association of a binding to a specific TD specification outside of the REC lifecycle of the given TD specification. This means that the process should outlive the WG if the WG ever comes to stop functioning or if the WG decides to not work on a TD specification version anymore.
 - There should be no two bindings for the same protocol, media type etc.
 - A binding should be identifiable by the elements in a form such as `href`, `contentType`, or other terms.
+  - We need to clarify whether the URI scheme and the media type MUST be registered in IANA first.
+    - Provisional registration could reduce the overhead. Any new conflicting ones would bring up a discussion but it can still result in "our" provisional getting demoted.
+    - Pros: more stable. Cons: More overhead and work
```

Comment:
Is the possibility to register something like a URI schema or a new media type within a new Binding Template already implied here? Because I think a Binding Template document could serve as a stable reference as required by IANA, so it could potentially fill this kind of gap (if that makes sense at all or shouldn't always be done by a different/the "original" SDO).

Reply:
My proposal going forward with this when it comes to that:
1. Try to involve the original SDO and get them to register the URI scheme
2a. If they are not responsive, let the entry submitter register the URI scheme as provisional after having a first reviewed draft. Then let them register it permanently once the registry entry is ready. Once it is in IANA, move registry entry to final status.
2b. If there are disagreements with the original SDO, keep the registry entry in a draft status.

In all cases, keep registry entry as draft until the URI scheme is registered at IANA. Ideally, I would not want to have an IANA registration requirement but it is too risky to not do that. Someone else can do that and our binding breaks... 

Case 2a can easily happen in old protocols like Modbus and 2b can happen when the original SDO had prior discussions but never resulted in a registration (MQTT has this issue I think...)